### ISSUE TITLE
Camera not granted permission

### ISSUE THREAD

**Issue by eduardsik** [bhowiebkr/laser-level-webcam#20]
Mac, M2, latest OS

I have this error while running from the terminal:
"qt.permissions: Could not find permission plugin for QCameraPermission. Please make sure you have included the required usage description in your Info.plist
Access to camera not granted"

But the app starts and the camera is visible, without picture. 

What shoud I do?

**Comment by eduardsik** [comment]
I tried on another mac and a same error. 

python version 3.9.17
I can record with ffmpeg without permission: ffmpeg -f avfoundation -framerate 30 -i "0" -target pal-vcd ./test.mpg

**Comment by bhowiebkr** [comment]
Yeah it seems to be a problem with Mac and trying to get access to the camera using PySide. I'm not sure I can help with this. Maybe see what Google says about camera access with PySide on a Mac.

**Comment by box185** [comment]
> Yeah it seems to be a problem with Mac and trying to get access to the camera using PySide. I'm not sure I can help with this. Maybe see what Google says about camera access with PySide on a Mac.

I installed this on Mac OS Monterey and had the same issue. I couldn't understand which application / library was trying to access the camera. It turned out that it was the "Terminal" application that needed permission.

I found this:

https://apple.stackexchange.com/questions/360851/add-access-to-the-macbook-camera-for-the-terminal-application

There are two ffmpeg commands that can be run in the terminal. This forces Mac OS to ask permission for the Terminal to access the camera(s). Then in Apple Menu / System Preferences / Security and Privacy / Camera, you should see the Terminal application with a check box. With that done, I was able to run the laser-level-webcam script and have access to the camera.
